pkg/monitor/nodedetails/audit_log_loadbalancer_summary.go

```diff
@@ -0,0 +1,89 @@
+package nodedetails
+
+import (
+	"strings"
+
+	auditv1 "k8s.io/apiserver/pkg/apis/audit/v1"
+)
+
+type LoadBalancerCheckSummary struct {
+	perNodeLoadBalancerCheckSummary map[string]*PerNodeLoadBalancerCheckSummary
+}
+
+type PerNodeLoadBalancerCheckSummary struct {
+	nodeName string
+
+	// this uses the external load balancer. If these are present more than 10s after termination started
+	// it means the LB is not set up well. If they are veryLate, then it's
+	// very bad.
```

Comment:
> more than 10s after termination started

can you clarify what you mean by `termination` here, is it when the apiserver process received the TERM signal and have initiated shutdown process? in that case 10s does not make sense, so i guess you are referring to when the apiserver has finished waiting for shutdown-delay-duration?

for the latter case, the requests arriving after the shutdown-delay-duration elapses may not be audited depending on the shutdown mode.